wftests/warrior_tests/testcases/common_actions_tests/tc_to_verify_get_time_stamp_and_time_delta.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" ?>
+<Testcase>
+	<Details>
+		<Name>tc_to_verify_get_time_stamp_and_time_delta</Name>
+		<Title>verify comparison operations</Title>
+		<Datatype>Custom</Datatype>
+		<default_onError action="next"/>
+		<Logsdir>Execution</Logsdir>
+                <InputDataFile>No_Data</InputDataFile>
+		<Resultsdir>Execution</Resultsdir>
+		<Date>2016-12-30</Date>
+		<Time>16:20</Time>
+		<Engineer>Anil Kumar</Engineer>
+	</Details>
+	<Steps>
+        <step Driver="common_driver" Keyword="get_current_timestamp" TS="1">
+			<Arguments>
+			</Arguments>
+			<onError action="next"/>
+			<Description>To get current timestamp</Description>
+			<Execute ExecType="Yes"/>
+			<context>positive</context>
+			<impact>impact</impact>
+		</step>
+
+		<step Driver="common_driver" Keyword="get_time_delta" TS="1">
+			<Arguments>
+                <argument name="start_time" value="2018-10-22 16:14:21"/>
```

Comment:
You need to pass the output of the keyword get_current_timestamp to get_time_delta.